docs/install_guides/2022.1.1/installing-openvino-from-archive-linux.md

```diff
@@ -0,0 +1,174 @@
+   ```sh
+   tar xf <archive name>.tgz -C <destination_dir>
+   ```
+   where the `<destination_dir>` is the directory that you extract OpenVINO files to. You're recommended to set it as:
+   * For root users or administrators: `/opt/intel/`
+   * For regular users: `/home/<USER>/intel/`
+
+If you forgot to set the directory in Step 5, you can then use `sudo mv <extracted_folder> /opt/intel` (for root users or administrators), or `mv <extracted_folder> /home/<USER>/intel/` (for regular users) to set that.
+
+For simplicity, it is useful to create a symbolink link:
+```sh
+ln -s /home/<USER>/intel/<extracted_folder> /home/<USER>/intel/openvino_2022
+```
+If such link already exists, remove the previous link with `rm /home/<USER>/intel/openvino_2022`.
+
+The `/opt/intel/openvino_<version>/` or `/home/<USER>/intel/openvino_<version>/` will be referred as the standard OpenVINO `<INSTALL_DIR>` in this document.
+
+The core components are now installed. Continue to the next section to install components.
+
+### <a name="set-the-environment-variables"></a>Step 2: Configure the Environment
+
+You must update several environment variables before you can compile and run OpenVINO™ applications. Set environment variables as follows:
+
+```sh
+source <INSTALL_DIR>/setupvars.sh
+```  
+
+If you have more than one OpenVINO™ version on your machine, you can easily switch its version by sourcing `setupvars.sh` of your choice.
+
+> **NOTE**: You can also run this script every time when you start new terminal session. Open `~/.bashrc` in your favorite editor, and add `source <INSTALL_DIR>/setupvars.sh`. Next time when you open a terminal, you will see `[setupvars.sh] OpenVINO™ environment initialized`. Changing `.bashrc` is not recommended when you have many OpenVINO™ versions on your machine and want to switch among them, as each may require different setup.
+
+The environment variables are set. Next, you can download some additional tools.
+
+### <a name="model-optimizer">Step 3 (Optional): Install Additional Components
+
+Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter can only be installed via PyPI. See [Install OpenVINO™ Development Tools](installing-model-dev-tools.md) for detailed steps.
+
+OpenCV is necessary to run demos from Open Model Zoo (OMZ). Some OpenVINO samples can also extend their capabilities when compiled with OpenCV as a dependency. You are recommended to build OpenCV from source using the [instructions on Github](https://github.com/opencv/opencv/wiki/BuildOpenCV4OpenVINO).
+
+### <a name="optional-steps"></a>Step 4 (Optional): Configure Inference on Non-CPU Devices
+
+@sphinxdirective
+.. tab:: GNA
+
+   To enable the toolkit components to use Intel® Gaussian & Neural Accelerator (GNA) on your system, follow the steps in :ref:`GNA Setup Guide <gna guide>`.
+   
+.. tab:: GPU
+
```

Comment:
Can GPU be selected by default? I suppose it's more common / highly used device than GNA

Reply:
Yes sure.